R/MsBackendDataFrame-functions.R

```diff
@@ -74,6 +74,15 @@ NULL
     NULL
 }
 
+.valid_intensity_mz_columns <- function(x) {
+    if (any(lengths(mz(x)) != lengths(intensity(x))))
+        "Length of mz and intensity values differ for some spectra"
+    else NULL
+}
+
+#' data types of spectraData columns
+#'
+#' @noRd
 .SPECTRA_DATA_COLUMNS <- c(
     msLevel = "integer",
     rt = "numeric",
```

Comment:
I would suggest time call this `rtime` rather than `rt`, as `rt` is a function to generate a random sample from a t distribution.

Reply:
agree